config/locales/ne.yml

```diff
@@ -0,0 +1,1338 @@
+    order_resumed:
+    order_state:
+      address: ठेगाना
+      awaiting_return: फिर्ताको कुर्दै
+      canceled: रद्द गरियो
+      cart: कार्ट
+      complete: पूर्ण
+      confirm: निश्चित गर्नुहोस्
+      considered_risky:
+      delivery: वितरण
+      payment: भुक्तानी
+      resumed: पुनःसुरु भयो
+      returned: फिर्ता
+    order_summary: अर्डर सारांश
+    order_sure_want_to: के तपाईं पक्का यो अर्डर %{event} गर्न चाहानुहुन्छ?
+    order_total: कुल अर्डर
+    order_updated: अर्डर अपडेट गरियो
+    orders: अर्डर
+    other_items_in_other:
+    out_of_stock: स्टक बाहिर
+    overview: विस्तृत सर्वेक्षण
+    package_from: बाट प्याकेज
+    pagination:
+      next_page: अर्को पाना »
+      previous_page: "« अघिल्लो पृष्ठ"
+      truncate: "…"
+    password: पासवर्ड
+    paste: टाँस्नुहोस्
+    path: बाटो
+    pay: तिर्नु
+    payment: भुक्तानी
+    payment_could_not_be_created:
+    payment_identifier:
+    payment_information: भुक्तान जानकारी
+    payment_method: भुक्तानी विधि
+    payment_method_not_supported:
+    payment_methods: भुक्तानी विधिहरू
+    payment_processing_failed: भुक्तानी प्रशोधन गर्न सकिदैन, कृपया तपाईंले प्रविष्ट गर्नुभएको विवरणहरू जाँच गर्नुहोस्
+    payment_processor_choose_banner_text: यदि तपाईंलाई भुक्तानी प्रोसेसर छनौट गर्न मद्दत चाहिएमा, कृपया भ्रमण गर्नुहोस्
+    payment_processor_choose_link: हाम्रो भुक्तानी पृष्ठ
+    payment_state: भुक्तानी अवस्था
+    payment_states:
+      balance_due: बाँकी रहेको
+      checkout: चेकआउट
+      completed: पूरा भयो
+      credit_owed: क्रेडिट तिर्नुपर्ने
+      failed: असफल
+      paid: भुक्तान गरियो
+      pending: फैसला बांकी राखेको
+      processing: प्रशोधन गर्दै
+      void: रद्द
+    payment_updated: भुक्तानी अपडेट भयो
+    payments: भुक्तानी
+    pending:
+    percent: प्रतिशत
+    percent_per_item: प्रति वस्तु प्रतिशत
+    permalink: Permalink
+    phone: फोन
+    place_order: प्लेस अर्डर
+    please_define_payment_methods: कृपया पहिले केहि भुक्तानी विधिहरू परिभाषित गर्नुहोस्।
+    populate_get_error: केहि गलत भयो। फेरि वस्तु थप्ने प्रयास गर्नुहोस्।
+    powered_by: द्वारा संचालित
+    pre_tax_amount:
+    pre_tax_refund_amount:
+    pre_tax_total:
+    preferred_reimbursement_type:
+    presentation: प्रस्तुतिकरण
+    previous: अघिल्लो
+    previous_state_missing:
+    price: मूल्य
+    price_range: मूल्य दायरा
+    price_sack: मूल्य बोरा
+    process: प्रक्रिया
+    product: उत्पादन
+    product_details: उत्पाद विवरण
+    product_has_no_description: यस उत्पादको कुनै विवरण छैन
+    product_not_available_in_this_currency: यो उत्पादन चयनित मुद्रामा उपलब्ध छैन।
+    product_properties: उत्पाद गुणहरू
+    product_rule:
+      choose_products: उत्पादनहरू छनौट गर्नुहोस्
+      label:
+      match_all: सबै
+      match_any: कम्तिमा एक
+      match_none:
+      product_source:
+        group: उत्पादन समूहबाट
+        manual: म्यानुअली छनौट गर्नुहोस्
+    products: उत्पादनहरू
+    promotion: पदोन्नति
+    promotion_action: पदोन्नति कार्य
+    promotion_action_types:
```

Comment:
@KiranPoudel98 please recheck all promotion related translations, meaning should not change at all